interview-questions.md

```diff
@@ -8,48 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
-
-Researched answer:
+Your answer: Javascript Syntax eXchange? I forget what the acronym means but it's what allows javascript to play with HTML's toys.
 
+Researched answer: Dang, I was close. The acronym is Javascript Syntax eXtension, or just Javascript XML. An interesting Syntactical difference would be the need to dance around Javascript's reserved keywords, like how "for" in HTMLspeak would be "HTMLfor".
```

Comment:
I love that you used the for keyword as an example because you searched this (as it wasn't brought up in class) and then you elaborated. Great job!